packages/utils/src/cda-converter/cda-conversion-testing-script.ts

```diff
@@ -0,0 +1,206 @@
+// The objective of these tests are to test two things
+// 1. That we generate valid CDA that our CDA->FHIR converter doesn't error on. We cant know if EPIC won't error on our CDA yet, but not erroring on our own CDA is a good start
+// 2. That our FHIR to CDA converter generates accurate CDA, such that when our CDA is converted back to FHIR, we get the same FHIR bundle as the original FHIR bundle. Obviously this introduces
+//    a dependency on another converter, but for now its the easiest approach since short of manually having exact string comparisons between two CDAs, we can't know if our CDA is accurate.
```

Comment:
nit: let's use long-form comments for these and add them after the imports, please:

```ts
/**
 * ...
 */
```

See examples on `packages/utils/src/customer-requests`